docs/project/contribution_tools.md

```diff
@@ -351,6 +352,33 @@ lldb bazel-bin/toolchain/carbon
 Any installed version of LLDB at least as recent as the installed Clang used for
 building should work.
 
+### Using LLDB from the command line
+
+We include launch commands for running lldb in VSCode in
+[`.vscode/lldb_launch.json`](/.vscode/lldb_launch.json). But it's also possible
+to run lldb from the command line.
+
+Put the following in a `.lldbinit` file, replacing `/path/to/carbon-lang` with
+the path to the root of the git checkout:
+
+```
+command script import external/+llvm_project+llvm-project/llvm/utils/lldbDataFormatters.py
+settings append target.source-map . /path/to/carbon-lang
+settings append target.source-map /proc/self/cwd /path/to/carbon-lang
```

Comment:
Does the example scripting in https://github.com/llvm/llvm-project/blob/main/lldb/examples/customization/pwd-cd-and-system/.lldbinit make it possible to check in a `.lldbinit`?

Reply:
In particular, maybe this could be doing something with `script` to get the git root (or directory of the `.lldbinit` file)?

Reply:
Ok yes, done! Thanks for the assistance.

Reply:
Oh, updating the docs...

Reply:
And done too.